creator/data_templates/mutations/data_template.py

```diff
@@ -0,0 +1,217 @@
+import graphene
+from graphql import GraphQLError
+from graphql_relay import from_global_id
+from django.conf import settings
+
+from creator.organizations.models import Organization
+from creator.data_templates.models import DataTemplate
+from creator.data_templates.nodes.data_template import DataTemplateNode
+
+
+class CreateDataTemplateInput(graphene.InputObjectType):
+    """Parameters used when creating a new data_template"""
+
+    name = graphene.String(
+        required=True, description="The name of the data_template"
+    )
+    description = graphene.String(
+        required=True, description="The description of the data_template"
+    )
+    icon = graphene.String(
+        description="Name of the Font Awesome icon to use when displaying the "
+        "template"
+    )
+    organization = graphene.ID(
+        required=True,
+        description="The organization that will own this data_template",
+    )
+
+
+class UpdateDataTemplateInput(graphene.InputObjectType):
+    """Parameters used when updating an existing data_template"""
+
+    name = graphene.String(description="The name of the data_template")
+    description = graphene.String(
+        description="The description of the data_template"
+    )
+    icon = graphene.String(
+        description="Name of the Font Awesome icon to use when displaying the "
+        "template"
+    )
+    organization = graphene.ID(
+        description="The organization that will own this data_template"
+    )
+
+
+class CreateDataTemplateMutation(graphene.Mutation):
```

Comment:
for consistency with past mutation we might not want to change this but I've found it is nice to group mutation by data type rather than action. So dataTemplateCreate, dataTemplateUpdate. It helps keeps all the dataTemplateActions together in the docs.

This idea comes from https://github.com/Shopify/graphql-design-tutorial/blob/master/TUTORIAL.md#input-structure-part-1

Reply:
Ooo I do like that. That shopify tutorial is cool. Hmm but I guess for consistency's sake I'll keep it the way it is